yt_dlp/utils/_utils.py

```diff
@@ -2436,7 +2436,7 @@ def __init__(self, ydl, info_dict):
     def parse_playlist_items(cls, string):
         for segment in string.split(','):
```

Comment:
maybe we should also this so that the error message below is always accurate
```suggestion
    def parse_playlist_items(cls, string):
        if string.startswith(',') or string.endswith(','):
           raise ValueError('There is an invalid leading comma or a trailing comma')
        for segment in string.split(','):
```